build.sbt

```diff
@@ -86,7 +93,7 @@ lazy val commonSettings = sharedCommonSettings ++
   addCompilerPlugins(vAll, "kind-projector") ++ Seq(
   scalacOptions ++= scalacAllOptions,
   parallelExecution in Test := false
-) ++ warnUnusedImport ++ unidocCommonSettings
+) /* ++ warnUnusedImport */ ++ unidocCommonSettings // spurious warnings from macro annotations expected
```

Comment:
Small point (again): locally this works for me enabled, perhaps worth re-adding?